TDD_Check/src/checkPackage/Check.java

```diff
@@ -0,0 +1,232 @@
+package checkPackage;
+
+public class Check {
+	
+	private double amount;
+	private String text = "";
```

Comment:
this is a String , all operation with String will be not so efficient is we are using JDK previous to 6 , if we want to be efficient we could use StringBuilder to all operations with string or StringBuffer if we need to overcome synchronization